### PR TITLE
Converted IHandler.Export to async.

### PR DESCRIPTION
**Pull request by dnduffy**

### CODE REVIEW

**Comment by dnduffy** [comment]
I signed it!

File: src/OpenCensus.Abstractions/Trace/Export/IHandler.cs [hunk 1]
```diff
@@ -27,6 +28,7 @@ public interface IHandler
         /// Exports the list of spans to the backend.
         /// </summary>
         /// <param name="spanDataList">Collection of spans to export.</param>
-        void Export(IEnumerable<ISpanData> spanDataList);
+        /// <returns>A <see cref="Task"/> representing the asynchronous export operation.</returns>
+        Task Export(IEnumerable<ISpanData> spanDataList);
```
Comment (SergeyKanzhelev) [inline src/OpenCensus.Abstractions/Trace/Export/IHandler.cs]: can you please follow naming convention and call method `ExportAsync`.

File: src/OpenCensus.Exporter.ApplicationInsights/Implementation/TraceExporterHandler.cs [hunk 2]
```diff
@@ -35,235 +36,238 @@ public TraceExporterHandler(TelemetryConfiguration telemetryConfiguration)
             this.telemetryClient = new TelemetryClient(telemetryConfiguration);
         }
 
-        public void Export(IEnumerable<ISpanData> spanDataList)
+        public async Task Export(IEnumerable<ISpanData> spanDataList)
         {
-            foreach (var span in spanDataList)
+            await Task.Run(async () =>
```
Comment (SergeyKanzhelev) [inline src/OpenCensus.Exporter.ApplicationInsights/Implementation/TraceExporterHandler.cs:41]: can you please confirm this is the only change in this file. GitHub failed to compare nicely

File: src/OpenCensus.Exporter.ApplicationInsights/Implementation/TraceExporterHandler.cs [hunk 3]
```diff
@@ -35,235 +36,238 @@ public TraceExporterHandler(TelemetryConfiguration telemetryConfiguration)
             this.telemetryClient = new TelemetryClient(telemetryConfiguration);
         }
 
-        public void Export(IEnumerable<ISpanData> spanDataList)
+        public async Task Export(IEnumerable<ISpanData> spanDataList)
```
Comment (SergeyKanzhelev) [inline src/OpenCensus.Exporter.ApplicationInsights/Implementation/TraceExporterHandler.cs]: rename to `ExportAsync` please

**Review by SergeyKanzhelev — CHANGES_REQUESTED** [review]
please rename methods

File: src/OpenCensus.Abstractions/Trace/Export/IHandler.cs (hunk 1 above)
Comment (dnduffy) [reply src/OpenCensus.Abstractions/Trace/Export/IHandler.cs]: done

File: src/OpenCensus.Exporter.ApplicationInsights/Implementation/TraceExporterHandler.cs (hunk 2 above)
Comment (dnduffy) [reply src/OpenCensus.Exporter.ApplicationInsights/Implementation/TraceExporterHandler.cs:41]: Correct, I have re-verified that this is the only change in this file.

File: src/OpenCensus.Exporter.ApplicationInsights/Implementation/TraceExporterHandler.cs (hunk 3 above)
Comment (dnduffy) [reply src/OpenCensus.Exporter.ApplicationInsights/Implementation/TraceExporterHandler.cs]: Done everywhere except unrelated comments/strings.

**Review by SergeyKanzhelev — APPROVED** [review]

**Pull request merged**